src/components/DataCollector/index.js

```diff
@@ -41,6 +39,6 @@ const createDataCollector = ({ eventManager }) => {
 
 createDataCollector.namespace = "DataCollector";
 createDataCollector.abbreviation = "DC";
-createDataCollector.configValidators = createConfigValidators();
+createDataCollector.configValidators = {};
 
```

Comment:
What is more readable?
This or
```
createDataCollector : {
  namespace: "DataCollector",
  abbreviation: "DC",
  configValidators: {}
}
}
```

Reply:
Doing it in an object like this is probably more readable, but I don't think there is a way to do it when you are adding properties to the function, and not just a generic object.  (createDataCollector is a function)